server/eventlog/eventlog.go

```diff
@@ -32,8 +33,16 @@ const (
 	numReadWorkers = 16
 )
 
-func GetEventLogPathFromInvocationId(invocationId string) string {
-	return invocationId + "/chunks/log/eventlog"
+var (
+	//Id of an empty log
+	EmptyId = chunkstore.ChunkIndexAsStringId(chunkstore.EmptyIndex)
```

Comment:
Yeah, it gets used in `build_event_handler.go`. I think it would also be nice to use it in the other places where I was doing this conversion manually, but I didn't want to include that in this CL.